src/controllers/index.ts

```diff
@@ -0,0 +1,8 @@
+export * from "./Account";
```

Comment:
I think exporting explicitly instead of * would be better here, since you are just instantly mapping over all exports from this file